extensions/amp-list/0.1/amp-list.js

```diff
@@ -514,24 +514,33 @@ export class AmpList extends AMP.BaseElement {
       (isFetch && this.element.hasAttribute('reset-on-refresh')) ||
       this.element.getAttribute('reset-on-refresh') === 'always'
     ) {
-      // Placeholder and loading don't need a mutate context.
-      this.togglePlaceholder(true);
-      this.toggleLoading(true, /* opt_force */ true);
-      this.mutateElement(() => {
-        this.toggleFallback_(false);
-        // Clean up bindings in children before removing them from DOM.
-        if (this.bind_) {
-          const removed = toArray(this.container_.children);
-          this.bind_.rescan(/* added */ [], removed, {
-            'fast': true,
-            'update': false,
+      let currentHeight;
+      this.measureMutateElement(
+        () => {
+          currentHeight = this.element./*OK*/ offsetHeight;
+        },
+        () => {
+          setImportantStyles(this.element, {
+            'height': `${currentHeight}px`,
+            'overflow': 'hidden',
           });
```

Comment:
Why do we need to lock height during reset?

Reply:
The list toggles the placeholder on when resetting. If this is a smaller height it causes content jumping.

Reply:
Oh good catch. I suppose we can handle height shrinking separately.